crates/search/src/buffer_search.rs

```diff
@@ -1075,6 +1075,12 @@ impl BufferSearchBar {
         }
     }
 
+    fn focus(&self, handle: &gpui::FocusHandle, cx: &mut ViewContext<Self>) {
+        cx.on_next_frame(|_, cx| {
```

Comment:
This seems to be out of date, as invalidate_character_coordinates already waits for the next frame.